src/dodal/beamlines/__init__.py

```diff
@@ -16,6 +16,7 @@
     "p47": "training_rig",
     "p48": "training_rig",
     "p49": "training_rig",
+    "t01": "adsim",
```

Comment:
Could: Do we still need this? It will make `dodal connect t01` redirect to `adsim` but does anyone know what `t01` is?

Reply:
I don't think it hurts to have it. t01 is used in the docs for the [example-services](https://github.com/epics-containers/example-services) that deploy the containerised IOCs.